recipes/_configure_indexes.rb

```diff
@@ -36,6 +36,20 @@
       :index
     end
 
+  daily_mb = hash.delete('_maxDailyDataSizeMB')
```

Comment:
Fair point.  Added in a configurable padding here: https://github.com/cerner/cerner_splunk/pull/123/commits/654b2d5bbbcbad2e48d2e172637bf4c1d5cd17f0